app/Controllers/Http/UserController.ts

```diff
@@ -52,6 +54,7 @@ const franzRequest = (route, method, auth) =>
 export default class UsersController {
   // Register a new user
   public async signup({ request, response, auth }: HttpContextContract) {
+    // @ts-ignore
```

Comment:
Always use `@ts-expect-error` over `@ts-ignore`, see more: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/prefer-ts-expect-error.md

Reply:
Totally agree, I'm changing it now!